packages/remix/src/generators/application/application.impl.ts

```diff
@@ -78,25 +79,7 @@ export default async function (tree: Tree, _options: NxRemixGeneratorSchema) {
     },
   });
 
-  const installTask = addDependenciesToPackageJson(
-    tree,
-    {
-      '@remix-run/node': remixVersion,
-      '@remix-run/react': remixVersion,
-      '@remix-run/serve': remixVersion,
-      isbot: isbotVersion,
-      react: reactVersion,
-      'react-dom': reactDomVersion,
-    },
-    {
-      '@remix-run/dev': remixVersion,
-      '@remix-run/eslint-config': remixVersion,
-      '@types/react': typesReactVersion,
-      '@types/react-dom': typesReactDomVersion,
-      eslint: eslintVersion,
-      typescript: typescriptVersion,
-    }
-  );
+  const installTask = updateDependencies(tree);
```

Comment:
We should still invoke the plugin `init` generator. It would cover the scenario of someone installing the plugin (without using `nx add`) and generating an application.

Additionally, we invoke this function that installs additional dependencies not installed by the `init` generator.